### ISSUE TITLE
Combined_Key error for Wayne Michigan starting on March 28th, until April 1st. 

### ISSUE THREAD

**Issue by wreganpdx** [CSSEGISandData/COVID-19#1891]
There is a key error, where there is no space between Wayne and Michigan.

**Comment by oltdaniel** [comment]
@wreganpdx Please provide an excact line in an file in this repo.

**Comment by wreganpdx** [comment]
Line 2118 
in file: 
COVID-19/csse_covid_19_data/csse_covid_19_daily_reports/04-01-2020.csv

Line 2066
in file:
https://github.com/CSSEGISandData/COVID19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/03-31-2020.csv

Line 3027
in file:
https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/03-29-2020.csv

Line 3031
in file
https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_daily_reports/03-30-2020.csv

**Comment by wreganpdx** [comment]
expected "Wayne, Michigan"
got "Wayne,Michigan"

Thank you

**Comment by oltdaniel** [comment]
@wreganpdx As an deveoper you need to be able to handle a space. Please close this issue.

**Comment by wreganpdx** [comment]
You realize there is an edit button for anyone who has permissions? Why should I fork the entire database, merge the changes, and then upload to fix a key error? Github has an edit button for stuff like this. Do the right thing. I will not close the issue.

**Comment by oltdaniel** [comment]
@wreganpdx If the data is very important to your application, you parse it your own onw and **store** it on your server. What ouye are talking about is an GitHub Repo to convert the data. That is stupid. Convert it with a good script by your own, thn use it with your app. No git command incvolved, just raw git cdn stuffl

**Comment by wreganpdx** [comment]
No it's not important to my application. I have just written a little app which isn't even on github yet because I'm a lazy developer. I actually have a lot of stuff on my plate right now, mostly do with non-computer science related stuff.

Anyways... 

They Combined_Key should be unique and consistent, that's the point of it. I've taken a database class and so I know that much. Hope this gets fixed. I fixed one file and created a pull request, but I think it has to be approved and then there are 3 more files. Hope this gets done for other people's sake and I hope any other errors that people bring up are met with more than this. I am not going to close this issue. If you have the authority to truly close the issue, you can close it yourself. If you don't have permission to close it, then it will remain open for someone more responsible than yourself to fix the issue or to close this themselves, if they think the error is unworthy of being fixed.

**Comment by oltdaniel** [comment]
Yes, @CSSEGISandData should keep the same CSV header names. But from my piont og view, they do an great job of providing the data, wether in a good format or not. Iths the reason why I convert it to per country data.

**Comment by wreganpdx** [comment]
Well, they could do a better job organizing the data, whatever they choose. I would suggest creating a "View" for countries if you use SQL. That's what Views are for.

**Comment by oltdaniel** [comment]
@wreganpdx Well, this data is only CSV no no SQL connection there. Everyone in SQL decides on a certain structure, that is mostly independetnt of this repo. And this SQL structure or any other kind of structure apart of this CSV one in here should be created by your own.
